test/integration/features/addon-xpaas.feature

```diff
@@ -0,0 +1,34 @@
+@addon-xpaas @addon
+Feature: XpaaS add-on
+
+  Scenario: User enables the xpaas add-on
+     When executing "minishift addons enable xpaas" succeeds
+     Then stdout should contain "Add-on 'xpaas' enabled"
+
+  Scenario: User starts Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+      And stdout should contain
+      """
+      XPaaS OpenShift imagestream and templates installed
+      """
+
+  Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
+   Given Minishift has state "Running"
+    When executing "oc new-app <template-name>" succeeds
```

Comment:
I don't understand this, how ```<template-name>``` is being parsed ?

Reply:
It is a so-called Secario Outline - https://github.com/cucumber/cucumber/wiki/Scenario-Outlines.

A scenario is repeated based on some variables defines in the examples table.